packages/web-components/fast-foundation/src/design-system/design-system.ts

```diff
@@ -76,6 +76,12 @@ export interface ElementDefinitionContext {
  */
 export type ElementDefinitionCallback = (ctx: ElementDefinitionContext) => void;
 
+interface ElementDefinitionParams
```

Comment:
I think we should probably export this and then add documentation comments. We'll also need the `@public` indicator in the comment to satisfy our docs generation stuff.

Reply:
Should be fixed now 👍